node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -1455,7 +1455,7 @@ async fn poll_collation_response(
 				);
 
 				CollationFetchResult::Error(COST_WRONG_PARA)
-			}
+			},
```

Comment:
These changes seem unrelated, can you revert them?

I suspect that your rustfmt might be outdated, you may check that by running `cargo fmt` on master.

Reply:
I use the latest rust version, these changes are present in master, the branch is simply outdated